CondTools/BeamSpot/python/BeamProfile2DBRead_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this file is not needed anymore, is it? (due to fillDesc generating it)

Reply:
For this config file the `cfi` is not automatically generated, so this one is needed.